content/projects/gsoc/2022/projects/jenkinsfile-runner-action-for-github-actions.adoc

```diff
@@ -50,7 +50,7 @@ It also provides one central documentation page and one demo repository to help
 There is a blog about the developing progress of these Jenkinsfile Runner Actions.
 It's useful for users to get started easier and learn their underlying principles.
 
-* link:https://deploy-preview-5402--jenkins-io-site-pr.netlify.app/blog/2022/09/07/jenkinsfile-runner-as-github-actions/[A near Feature-complete version of Jenkinsfile Runner Actions as GitHub Actions]
+* link:https://www.jenkins.io/blog/2022/09/07/jenkinsfile-runner-as-github-actions/[A near Feature-complete version of Jenkinsfile Runner Actions as GitHub Actions]
 
```

Comment:
This syntax breaks the build.
It is recommended to use `link:/blog/2022/09/07/jenkinsfile-runner-as-github-actions/[A near Feature-complete version of Jenkinsfile Runner Actions as GitHub Actions]`

Reply:
I see. I just fix this error.